packages/gpu-operator/v24.3.0+1/package.yaml

```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "gpu-operator"
+shortDescription: >-
+  The GPU Operator allows administrators of Kubernetes clusters to manage GPU nodes just like CPU nodes in the cluster.
+iconUrl: "https://avatars.githubusercontent.com/u/1728152"
+defaultNamespace: "gpu-operator"
+helm:
+  repositoryUrl: "https://helm.ngc.nvidia.com/nvidia"
+  chartName: "gpu-operator"
+  chartVersion: "v24.3.0"
+dependencies:
+  - name: node-feature-discovery
+references:
+  - label: ArtifactHub
+    url: 
```

Comment:
empty `url` is not allowed in the schema. If there is no artifacthub URL, please remove the entry